### PR TITLE
feat: improvement in the pseudo-class: focus

### PR DESCRIPTION
**Pull request by GusBaamonde**
Improvement in the pseudo-class: focus to be able to navigate the forms with tab and improve our accessibility

focus-form

#Checklist

- [X] Check if there is a new color.
- [X] Create the color variable in the _colors.scss file.
- [X] Follow the color nomenclature.
- [X] Create the color class with its variable.
- [X] Create the color component in the html, show the name, color class and its exadecimal.

Look: http://cdn.fromdoppler.com/doppler-ui-library/build871/signup.html

focus-buttons

### CODE REVIEW

File: assets/scss/modules/_buttons.scss [hunk 1]
```diff
@@ -160,4 +160,9 @@
 }
 .dp-library {
   @include dp-button();
+
+  button:focus {
+    outline: none;
+    box-shadow: inset 0 0 0 1px #e79433, inset 0 0 0 2px #ffffff;
```
Comment (cbernat) [inline assets/scss/modules/_buttons.scss:166]: Here please remember to add variables for the colors here,

Comment (cbernat) [reply assets/scss/modules/_buttons.scss:166]: Why move this focus inside here and not inside `dp-button` mixin?

File: assets/scss/modules/_forms.scss [hunk 2]
```diff
@@ -206,11 +202,17 @@
...
-    opacity: 0;
-    z-index: 0;
     cursor: pointer;
-    zoom: 2.5;
     position: absolute;
+    width: 14px;
+    height: 14px;
+
+    &:focus {
+      outline: 4px solid #e79433;
```
Comment (cbernat) [inline assets/scss/modules/_forms.scss:211]: Here you've miss a variable for color `#e79433`

File: assets/scss/modules/_forms.scss [hunk 3]
```diff
@@ -550,4 +552,12 @@
       left: 9px;
     }
   }
+
+  select:focus,
+  textarea:focus,
+  input:focus {
+    outline: none;
+    border: 1px solid #e79433;
```
Comment (cbernat) [inline assets/scss/modules/_forms.scss:560]: Same here for `#e79433`

File: assets/scss/modules/_forms.scss (hunk 2 above)
Comment (andresmoschini) [inline assets/scss/modules/_forms.scss:211]: I understand that the solution for the _checkbox_ is different than in other _inputs_, but, what do you think about to keep it closer? I mean, in place of declare the style here, do it near the other focus declarations:

```sass
  select:focus,
  textarea:focus,
  input:focus {
    outline: none;
    border: 1px solid #e79433;
    box-shadow: inset 0 0 0 2px #e79433;
  }

   input[type='checkbox']:focus {
      outline: 4px solid #e79433;
      outline-offset: 0px;
      top: 1px;
      left: 1px;
    }
```

**Comment by andresmoschini** [comment]
Great work! very polished!

I have a question regarding the behavior. Shold the focus not mark the checkbos and the label?

image

**Comment by cbernat** [comment]
❤️

**Pull request merged**